datalad/crawler/nodes/annex.py

```diff
@@ -1327,7 +1327,7 @@ def _finalize(data):
                         tag_ = last_version
                     # TODO: config.tag.sign
                     stats_str = "\n\n" + total_stats.as_str(mode='full')
-                    tags = self.repo.repo.tags
+                    tags = [t['name'] for t in self.repo.get_tags()]
```

Comment:
Getting the names of the tags seems to be too common to repeat as a list comprehension... Should get_tags get an argument to just return names?